src/driver/franka_plan_runner.cc

```diff
@@ -848,6 +851,23 @@ franka::JointPositions FrankaPlanRunner::JointPositionCallback(
   }
   output_to_franka = utils::EigenToArray(output_to_franka_eigen);
 
+  if (plan_exec_opt_ == robot_msgs::plan_exec_opts_t::MOVE_UNTIL_STOP) {
+    Eigen::Map<const Eigen::Matrix<double, 6, 1>> cartesian_contact(
+        robot_state.cartesian_contact.data());
+
+    const auto expected_contact_bool {contact_expected_.array() > 0};
+    const auto contact_bool {cartesian_contact.head(3).array() > 0};
```

Comment:
I worry that this might be nonzero sometimes when not actually in contact, like for example when running with garment. Have we verified this?

Reply:
Yeah, we tested this on jello yesterday. Also, it only has two states: 0 and 1 based on the contact threshold

Reply:
The `> 0` is really just to convert `Eigen<double>`  to `Eigen<bool>`